content/post/dataproc_lzo.md

```diff
@@ -0,0 +1,51 @@
+---
+author: vadim
+date: 2016-10-21
+title: "Adding LZO support to Dataproc"
+draft: false
+image: /post/dataproc_lzo/intro.png
+description: "How to leverage splittable LZO compression in Dataproc"
+---
+[LZO](https://en.wikipedia.org/wiki/Lempel%E2%80%93Ziv%E2%80%93Oberhumer) is a nice
+general-purpose data compression algorithm which provides blazing fast decompression.
+It is especially good because LZO archives can be indexed and become
+[splittable](http://stackoverflow.com/questions/34208051/when-are-files-splittable).
+source{d} uses LZO to store the dataset of names inside GitHub repositories,
+reducing the amount of data by a factor of 2.
+
+Spark does not support LZO codec out of the box because it's implementation
+is GPL licensed and conflicts with Apache. Google's [Dataproc](https://cloud.google.com/dataproc/),
+being the managed Spark cluster cloud solution, obviously does not support LZO out of the box too.
+However, Twitter have developed
```

Comment:
I meant Twitter as a group of people. http://english.stackexchange.com/questions/1338/are-collective-nouns-always-plural-or-are-certain-ones-singular
This is because I was taught British English and you - American :)